### PR TITLE
Fix for display issues in operator hub details

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes text overrun in the left side properties on Firefox
Loads correct styles for the description markdown text

Fixes https://jira.coreos.com/browse/CONSOLE-1209

### CODE REVIEW

**Comment by serenamarie125** [comment]
can you add a screenie for good measure @jeff-phillips-18

**Comment by jeff-phillips-18** [comment]
image

File: frontend/public/components/catalog/_catalog.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@ $catalog-item-icon-size-sm: 24px;
   @include co-break-word;
 }
 
+.properties-side-panel-pf-property-value {
```
Comment (rhamilto) [inline frontend/public/components/catalog/_catalog.scss]: Nit:  combine with above rule since the declarations are the same?

**Pull request merged**